ui/guest/src/store/util.ts

```diff
@@ -72,7 +72,7 @@ export function beforeWrite$<T extends any = 'continue', S extends any = never>(
           tap(({ payload }) => payload.type !== 'continue' && post(unlockItem({ path }))),
           switchMap(({ payload }) => (payload.type === 'continue' ? continue$ : stop$))
         );
-      } else if (item.commitId !== localItem.commitId && item.lockOwner !== username) {
+      } else if (item.commitId !== localItem?.commitId && item.lockOwner !== username) {
```

Comment:
Do we actually want an out of sync if the local item is null? Doesn't sound right.